src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -648,6 +648,77 @@
     .power 10
     $.equal-to 2655.992279142402
 
+[] > sqrt-check-zero-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          0.0
+          sqrt.
+            number 0.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-negative-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number -0.1
+    $.equal-to TRUE
+
+[] > sqrt-check-float-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          2.0
+          sqrt.
+            number
+              4.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-int-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          9.0
+          sqrt.
+            number
+              81
+    $.less-than 0.00000000001
+
+[] > sqrt-check-nan-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number nan
+    $.equal-to TRUE
+
+[] > sqrt-check-max-int
+  9223372036854775807 > max-int!
+  assert-that > @
+    abs.
+      number
+        minus.
+          3037000499.97605
+          sqrt.
+            number
+              max-int
+    $.less-than 0.00001
```

Comment:
@levBagryansky why have you made precision value equal to `0.0001`, but not to `0.00000000001` as it was in other tests?

Reply:
@includealex because there is a big float here